app/models/github/issue_label.rb

```diff
@@ -3,4 +3,37 @@ class Github::IssueLabel < ApplicationRecord
     inverse_of: :labels,
     foreign_key: "github_issue_id",
     class_name: "Github::Issue"
+
+  def self.type_prefix(type)
+    "x:#{type}"
+  end
+
+  def self.for_type(type, val)
+    return unless TYPES.include?(type)
+    return unless TYPES[type].include?(val)
+
+    "#{type_prefix(type)}/#{val}"
+  end
+
+  def of_type?(type)
+    name.start_with?(Github::IssueLabel.type_prefix(type))
+  end
+
+  def prefix
```

Comment:
May as well memoize this and value below. Save us a little bit of allocation in the commands where we repeatidly iterate through the label names/values.